### PR TITLE
List (mobile): Update List examples to apply changed UX

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] N/A
[Problem] After changing the UX, existing sreen of exmaples is weird.
[Solution] Apply the changed UX to examples(on-off,selectall, reorder, and expandable)

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by HunseopJeong** [comment]
On-off elements
image
Selectable
image
Reorder
image
image

**Comment by TomaszLukawskiSam** [comment]
@lslachciak @HunseopJeong 
This PR looks good to me but we have to consider which current listview examples we need in UIComponents app and which examples should be removed or updated.

**Comment by HunseopJeong** [comment]
Yes. We need to consider which examples should be updated or deleted.
There are so many duplicated examples and outside of scope of the UX guide.

File: examples/mobile/UIComponents/components/list/reorder.html [hunk 1]
```diff
@@ -229,16 +224,14 @@ <h1>
...
-							<span>
-								Primary text
-							</span>
+				<li>
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Multi primary text
 						</span>
-						<span class="ui-li-text-2">
+						<span class="ui-li-text-sub ui-li-text-value">
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/list/reorder.html:234]: Do you use sub text instead of 2?
If so, how do you handle cases of 3 or higher?

**Review by DohyungLim — COMMENTED** [review]
I think that code other than 1 comment looks fine.

File: examples/mobile/UIComponents/components/list/reorder.html (hunk 1 above)
Comment (HunseopJeong) [reply examples/mobile/UIComponents/components/list/reorder.html:234]: I changed original usages.
We can place the text to the correct position by using the 'ui-li-text-title' and 'ui-li-text-sub'.

**Review by singa2000 — APPROVED** [review]
Now it  looks  good.
If it's needed, other samples will be update in later.

**Pull request merged**